provider/database.py

```diff
@@ -17,15 +17,30 @@
 import uuid
 
 from cloudant import Cloudant
+from cloudant import CouchDB
 from cloudant.result import Result
 
 class Database:
     db_prefix = os.getenv('DB_PREFIX', '')
     dbname = db_prefix + 'ow_kafka_triggers'
-    username = os.environ['CLOUDANT_USER']
-    password = os.environ['CLOUDANT_PASS']
+    username = os.environ.get('CLOUDANT_USER')
+    client = None
 
-    client = Cloudant(username, password, account=username)
+    if (username):
```

Comment:
Also, why keep `CLOUDANT_USER` _and_ `DB_USERNAME`? Seems like we only need one of these.